draft-issuance.md

```diff
@@ -0,0 +1,162 @@
+scheme would approximate the current issuance over 4-year intervals.
+
+# Motivation
+
+The current Zcash economic model, inherited from Bitcoin, includes a halving mechanism which dictates the issuance of new coins. While this has been foundational, halvings can lead to abrupt changes in the rate of new coins being introduced to the market. Such sudden shifts can potentially disrupt the network's economic model, potentially impacting its security and stability.
+
+To address this, we propose smoothing out the issuance curve of ZEC, ensuring a more consistent and predictable rate of coin issuance, while still preserving the overall supply cap of 21,000,000 coins. This modification does not seek to change the core aspects of Zcash's issuance policy. Instead, it solely aims to enhance predictability and avoid sudden changes. By making this shift, the average block subsidy over time will remain consistent, and the funds for block subsidies will be available for a similar duration.
+
+Additionally, the current Bitcoin-style issuance does not take into account the current balance of `ZsfBalanceAfter(h)`. If [#draft-zsf]_ were to activate without a change to the issuance mechanism, then some funds would never be disbursed after they are deposited back into the ZSF.
+
+In summary, by introducing a smoother emissions curve, we:
+- maintain the economic viability of Zcash
+- provide clarity on distribution of deposits into the ZSF
+- enhance Zcash's stability as the network evolves.
+
+# Requirements
+
+Smoothing the issuance curve is possible using an exponential decay formula that
+satisfies the following requirements:
+
+## Issuance Requirements
+
+1. The issuance can be summarised into a reasonably simple explanation
+2. Block subsidies approximate a continuous function
+3. If there are funds in the ZSF, then the block subsidy must be non-zero, preventing any final “unmined” zatoshis
+4. For any 4 year period, all paid out block subsidies are approximately equal to half of the ZSF at the beginning of that 4 year period, if there are no deposits into the ZSF during those 4 years
+TODO daira: add a requirement that makes the initial total issuance match the previous total issuance  
+5. This functionality MUST be introduced as part of a network upgrade
+
+
+# Specification
+
+## Constants
+
+Define constants:
+
+“`BLOCK_SUBSIDY_FRACTION`” = 41 / 100,000,000 or `0.00000041`
+
+## Issuance Calculation
+
+Given the block height `h` define a function **BlockSubsidy(h)**, such that:
+
+**BlockSubsidy(h)** = Block subsidy for a given `h`, that satisfies above requirements.
+
+Using an exponential decay function for **BlockSubsidy** satisfies requirements **R1** and **R2** above:
+
+`BlockSubsidy(h) = BLOCK_SUBSIDY_FRACTION * ZsfBalanceAfter(h - 1)`
+
+Finally, to satisfy **R3** above we always round up to the next zatoshi.
+
+`BlockSubsidy(h) = ceiling(BLOCK_SUBSIDY_FRACTION * ZsfBalanceAfter(h - 1))`
```

Comment:
After the `Issuance Calculation` section, the specification needs a section where it specifies how this is intended to be "connected" to the previous issuance mechanism. This is important because it defines the shape of the issuance curve at the transition (in the currently-included image this is a jump upwards).

There will be some block height (that should be a constant in this specification section with either a known value or undefined for now) at which this issuance mechanism will first be applied, but that does not necessarily need to coincide with the network upgrade activation height (so e.g. if this were deployed in a NU that activated at height `X`, the height at which the issuance switches from the old to the new mechanism can be any height `Y >= X`).

There should also be a section in the Rationale explaining why the (to-be-specified) choice was made.

Reply:
@nathan-at-least This is what we spoke about on our recent call - can you weigh in here?

Reply:
@str4d I believe I've now addressed this in my latest commit(s) based on the discussion on the Oct 5 Arborist call - can you confirm?